src/axom/inlet/Table.cpp

```diff
@@ -91,39 +112,48 @@ std::vector<VariantKey> Table::containerIndices() const
       auto view = group->getView(idx);
       if(view->getTypeID() == axom::sidre::CHAR8_STR_ID)
       {
-        indices.push_back(view->getString());
+        std::string string_idx = view->getString();
+        VariantKey key = string_idx;
+        if(trimAbsolute)
+        {
+          // If the index is full/absolute, we only care about the last segment of it
+          string_idx = removeBeforeDelimiter(string_idx);
+          // The basename might be an integer, so check and convert accordingly
+          int idx_as_int;
+          if(checkedConvertToInt(string_idx, idx_as_int))
+          {
+            key = idx_as_int;
+          }
+          else
+          {
+            key = string_idx;
+          }
+        }
+        indices.push_back(key);
       }
       else
       {
         indices.push_back(view->getData<int>());
       }
     }
   }
-  else
-  {
-    SLIC_ERROR(
-      fmt::format("[Inlet] Table '{0}' does not contain an array or dict",
-                  m_name));
-  }
   return indices;
 }
 
 std::vector<std::pair<std::string, std::string>> Table::containerIndicesWithPaths(
   const std::string& name) const
 {
   std::vector<std::pair<std::string, std::string>> result;
-  // Need to go up one level because this is an _inlet_container group
-  const auto pos = m_name.find_last_of("/");
-  const std::string baseName = m_name.substr(0, pos);
-  for(const auto& indexLabel : containerIndices())
+  for(const auto& indexLabel : containerIndices(false))
```

Comment:
Side note: It seems like a lot of this logic would be simplified with `sidre` (or possibly axom `core`) had a `Path` class to extract the stem/root etc...  (and to support different delimiters).